packages/back-end/src/services/features.ts

```diff
@@ -138,13 +151,45 @@ export function generateFeaturesPayload({
       groupMap,
       experimentMap,
       safeRolloutMap,
+      holdoutsMap,
     });
     if (def) {
-      defs[feature.id] = def;
+      featureDefs[feature.id] = def;
     }
   });
+  holdoutsMap.forEach((holdout) => {
+    const exp = holdout.experiment;
+    if (!exp) return;
 
-  return defs;
+    const def: FeatureDefinitionWithProjects = {
+      defaultValue: "genpop",
+      rules: [
+        {
+          id: getHoldoutFeatureDefId(holdout.id),
+          coverage: exp.phases[0].coverage,
+          hashAttribute: exp.hashAttribute,
+          seed: exp.phases[0].seed,
```

Comment:
Hardcoding this to the first phase feels a little fragile.  At least add a comment here